### PR TITLE
Add rolldown force tests

### PR DESCRIPTION
**Pull request by AlexChetverov**

### CODE REVIEW

File: test/sequential/rolldown.forceUpdate.test.ts [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+async function setupASequencer(user: User, chain: ChainName = "Ethereum") {
+  const extrinsic = await SequencerStaking.provideSequencerStaking(
+    (await SequencerStaking.minimalStakeAmount()).addn(1000),
+    chain,
+  );
+  await Sudo.batchAsSudoFinalized(
+    Assets.mintNative(user),
+    Sudo.sudoAs(user, extrinsic),
+  );
+}
```
Comment (goncer) [inline test/sequential/rolldown.forceUpdate.test.ts:47]: Is not elsewhere a function that does exactly that?

File: test/sequential/rolldown.forceUpdate.test.ts [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+beforeEach(async () => {
+  //TODO: Replace this by some monitoring of the active queue.
+  const activeSequencers = await SequencerStaking.activeSequencers();
+  for (const chain in activeSequencers.toHuman()) {
+    for (const seq of activeSequencers.toHuman()[chain] as string[]) {
+      if (seq !== null) {
+        await leaveSequencing(seq);
+      }
+    }
+  }
```
Comment (goncer) [inline test/sequential/rolldown.forceUpdate.test.ts:66]: same here, I think you already refactored this  some time ago.

**Review by goncer — CHANGES_REQUESTED** [review]

File: test/sequential/rolldown.forceUpdate.test.ts (hunk 1 above)
Comment (AlexChetverov) [reply test/sequential/rolldown.forceUpdate.test.ts:47]: No, the another function doesn't mint token for user
We have 3 ways how to solve it
1)Keep the function
2)Add the function to SequencerStaking.ts
3)Delete the function and add 2 line instead: minting and provideSequencerStaking

File: test/sequential/rolldown.forceUpdate.test.ts (hunk 2 above)
Comment (AlexChetverov) [reply test/sequential/rolldown.forceUpdate.test.ts]: Added the function

File: test/sequential/rolldown.forceUpdate.test.ts (hunk 1 above)
Comment (goncer) [reply test/sequential/rolldown.forceUpdate.test.ts:47]: then is fine, no worries, you can leave it here,
ty and sorry!

**Review by goncer — APPROVED** [review]

**Pull request merged**